packages/flutter_tools/test/general.shard/base/common_test.dart

```diff
@@ -23,5 +24,9 @@ void main() {
     test('throws ToolExit with message and exit code', () {
       expect(() => throwToolExit('message', exitCode: 42), throwsToolExit(exitCode: 42, message: 'message'));
     });
+
+    testWithoutContext('Throws if accessing the Zone', () {
+      expect(() => fs, throwsA(isInstanceOf<UnsupportedError>()));
```

Comment:
Should this test just do a `context.get()` directly so that it isn't dependent on how the `fs` getter is implemented? A separate test could check that `fs` checks the `context`, but I'm not sure how valuable that test would be.

Reply:
Done